app/build.gradle

```diff
@@ -95,6 +95,8 @@ dependencies {
     implementation 'com.daimajia.swipelayout:library:1.2.0@aar'
     //metadata extractor
     implementation 'com.drewnoakes:metadata-extractor:2.11.0'
+
+    compile group: 'javax.annotation', name: 'javax.annotation-api', version: '1.3.1'
```

Comment:
why is this needed?

Reply:
I was just trying to test things around. You are right, these were unnecessary commits I'll remove them.